automation_testing/tests/Components/dataview/data_view.spec.ts

```diff
@@ -1,48 +1,67 @@
-import { test, expect } from "@playwright/test";
+import { test, expect, Page } from "@playwright/test";
 import { ColumnsComponent } from "../../../pages/Components/DataView/ColumnsComponent";
 import { DataviewPage } from "../../../pages/Components/DataView/DataViewPage";
 import { dataview_data } from "../../../utils/data/dataview_data";
 
-test.describe("Data View", () => {
+test.describe.parallel("Components - Data View", () => {
+	let page: Page;
 	let dataviewPage: DataviewPage;
 	let columns: ColumnsComponent;
 
-	test.beforeEach(async ({ page }) => {
+	test.beforeAll(async ({ browser }) => {
+		page = await browser.newPage();
 		dataviewPage = new DataviewPage(page);
 		columns = dataviewPage.columnsComponent;
-
 		await dataviewPage.visitPage();
 	});
 
-	test("Create New", async () => {
+	test.beforeEach(async() => {
+		await page.reload();
+	});
+
+	test.afterAll(async ({ browser }) => {
+		browser.close;
+	});
+
+	test("Create New", async ({ page }) => {
+		page.on("dialog", async dialog => {
```

Comment:
I think this dialog listener can be reduced to a function that just gets called with the message it might expect. We were previously doing something similar with "setDialogValidationListener" is there a reason we stopped using that?

Reply:
This method was causing some issues in the test. For example, if I ran all the tests, the "Edit Icon" test will fail because the expected value is "Create New". I don't know why is taking the expected value of the previous test, that's why I chose to give each test its own dialog method.